WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -317,36 +315,14 @@ final class DashboardViewModel {
     /// Sets up observer to decide Blaze campaign view visibility
     ///
     private func setupObserverForBlazeCampaignView() {
-        guard featureFlagService.isFeatureFlagEnabled(.optimizedBlazeExperience) else {
-            return
-        }
-
         blazeCampaignDashboardViewModel.$shouldShowInDashboard
             .assign(to: &$showBlazeCampaignView)
     }
 }
 
-// MARK: - Blaze banner
-extension DashboardViewModel {
-    /// Checks for Blaze eligibility and user defaults to show the banner if necessary.
-    ///
-    @MainActor
-    func updateBlazeBannerVisibility() async {
-        showBlazeBanner = await isBlazeBannerVisible()
-    }
-
-    private func isBlazeBannerVisible() async -> Bool {
-        async let isSiteEligible = blazeEligibilityChecker.isSiteEligible()
-        async let storeHasPublishedProducts = (try? checkIfStoreHasProducts(siteID: siteID, status: .published)) ?? false
-        async let storeHasAnyOrders = (try? checkIfStoreHasOrders(siteID: siteID)) ?? false
-        guard await(isSiteEligible, storeHasPublishedProducts, storeHasAnyOrders) == (true, true, false) else {
-            return false
-        }
-        return !userDefaults.hasDismissedBlazeBanner(for: siteID)
-    }
-
+private extension DashboardViewModel {
     @MainActor
-    private func checkIfStoreHasProducts(siteID: Int64, status: ProductStatus? = nil) async throws -> Bool {
+    func checkIfStoreHasProducts(siteID: Int64, status: ProductStatus? = nil) async throws -> Bool {
```

Comment:
Super nit: we should update the error log to not specify Blaze banner to avoid confusion (posting here because I cannot comment on the line below 😅)

Reply:
Good point. Updated the error message in 6d15e77a35ae44748db948f3f8cadb957072dfe8